src/linker/Linker.Dataflow/DynamicallyAccessedMembersBinder.cs

```diff
@@ -298,5 +310,22 @@ public static IEnumerable<EventDefinition> GetEventsOnTypeHierarchy (this TypeDe
 				onBaseType = true;
 			}
 		}
+
+		public static IEnumerable<InterfaceImplementation> GetAllInterfaceImplementations (this TypeDefinition type, LinkContext context)
```

Comment:
Is it expected that it could return the same interface multiple times?

Reply:
You could consider moving this to TypeDefinitionExtensions

Reply:
Yes - it's OK if it returns the same interface multiple times. It was not worth optimizing (MarkType will almost immediately return if the type is already marked so adding another layer of cache here is not worth it).

For this reason I don't want to move this to TypeDefinitionsExtensions - it's not "Clean" - but for this purpose it doesn't have to be.